### DISCUSSION TITLE
Reporting Bugs

### DISCUSSION THREAD

**Discussion by blackgrease** [blockscout/blockscout#9826] [Support] [answered]
Hello, where can I report security bugs to?

**Comment by GimluCom** [comment]
https://github.com/blockscout/blockscout/security

**Reply by blackgrease** [reply]
I have reached out but there is no response. Its been a while thats why wondering if there was another communication channel

**Reply by GimluCom** [reply, marked as answer]
You should have or will receive a response soon. Please close if you already have received a reply.

**Reply by blackgrease** [reply]
> You should have or will receive a response soon. Please close if you already have received a reply.

Hello. I suspect your working with Blockscout team. There are a couple of reports previous submitted more than a month ago and neither have gotten any response. Could you kindly assist?
